Makefile

```
@@ -320,6 +319,12 @@ kind:
 	@test -s $(KIND) && $(KIND) --version | grep -q $(KIND_VERSION) || \
 	$(call go-install-tool,$(KIND),sigs.k8s.io/kind/cmd/kind@$(KIND_VERSION))
 
+HELM         := $(LOCALBIN)/helm
+HELM_VERSION := v3.17.1
```

```suggestion
HELM         := $(LOCALBIN)/helm
HELM_VERSION := v3.17.1
HELM_LOOKUP :=  helm/helm
```

This way renovate will be able to patch